CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [0.36.2] - 2024-01-15
+
+### Changes
+
+-   Add `dateProvider` as a config option aimed at resolving clock skew issues.
```

Comment:
```suggestion
-    Exporting the `DateProvider` from supertokens-web-js, that both built-in and custom validators can use instead of `Date.now` to get an estimate of the server clock.
-    Added the `dateProvider` prop to the configuration that can be used to customize the built-in `DateProvider`.
-    Added `getClockSkewInMillis` as an overrideable function to the Session recipe that estimates the time difference between the backend and the client.
```